### DISCUSSION TITLE
Disable code search keyboard shortcut.

### DISCUSSION THREAD

**Discussion by ItsIgnacioPortal** [community/community#29860] [Other Feature Feedback, Questions, & Ideas]
I want to disable the CTRL + F keyboard shortcut. It's very frustrating to have a new page open when I'm trying to **_quickly_** search for content in a long Issue or wiki page. I've even tried to completly opt out of this "technology preview" from the "feature preview" menu (click on your profile picture on the top right, then on "feature preview"), but the option just isn't there.

In fact, I think this "shortcut" should be disabled by default. It doesn't make sense for this shortcut to be enabled when not viewing a file. Code Search doesn't support issues, wiki pages, nor the account settings page.

It just doesn't sit well with me that a website tried to _hijack_ a keyboard shortcut from the browser.

**Comment by HiBixby** [comment]
Hello!😊
Is this what you looking for?
You can disable <kbd>Ctrl</kbd>+<kbd>F</kbd> in [here](https://github.com/settings/accessibility)
Or you can change shorcut config
I hope this helps you

**Reply by ItsIgnacioPortal** [reply]
Hello!

Well, I unmarked that checkbox months ago. Yet, for some reason, the shortcut seems to start working again sporadically. Every time I look, that checkbox is unchecked. I don't know what to make of it

**Reply by Mailaender** [reply]
That is only for
> [GitHub shortcuts](https://docs.github.com/en/get-started/using-github/keyboard-shortcuts) that don't use modifier key

so it has no effect on <kbd>CTRL</kbd> + <kbd>F</kbd>

**Reply by koponk04** [reply]
so we just give up huh? ok,,,

**Comment by alper** [comment]
This "feature" is immensely annoying and makes Github unusable for me for common day to day documentation reading.
